crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -824,10 +900,12 @@ pub struct ExecutionArgs {
     /// one-at-a-time) execution.
     #[clap(long)]
     pub concurrency: Option<String>,
-    /// Continue execution even if a task exits with an error or non-zero
-    /// exit code. The default behavior is to bail
-    #[clap(long = "continue")]
-    pub continue_execution: bool,
+    /// Specify which tasks should continue running when an error occurs.
+    /// Use "none" to cancel all tasks.
+    /// Use "independent-tasks-only" to continue running independent tasks and
+    /// cancel dependent ones. Use "all" to continue running all tasks.
+    #[clap(long = "continue", value_name = "CONTINUE", num_args = 0..=1, default_value = "none", default_missing_value = "all", value_parser = ContinueModeParser::new())]
```

Comment:
With not needing to support `true`/`false` we can just use `EnumValueParser`
```suggestion
    #[clap(long = "continue", value_name = "CONTINUE", num_args = 0..=1, default_value = "none", default_missing_value = "all", value_parser = EnumValueParser::<ContinueMode>::new())]
```

Reply:
was able to omit the explicit value_parser altogether, since afaict the ValueEnum stuff is automatic